content/en/docs/pipelines/tutorials/sdk-examples.md

```diff
@@ -82,3 +82,7 @@ find a pipeline's ID:
 <img src="/docs/images/sdk-examples-snapshot-1.png"
 alt="Pipeline ID in Summary Card"
 class="mt-3 mb-3 border border-info rounded">
+
+## Creating a run from a pipeline version using the SDK
+
+An example of creating a run from a pipeline version is at [Kubeflow Pipelines Github repository](https://github.com/kubeflow/pipelines/blob/master/tools/benchmarks/run_service_api.ipynb).
```

Comment:
Out of curiosity, why do you recommend creating ApiResourceReferences to specify the version instead of using the [run_pipeline method of kfp.Client](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.client.html#kfp.Client.run_pipeline)? This method has a parameter for the version id which seems like it would make it easier to use.

Reply:
create_run method in SDK is an direct mirror of CreateRun method in our api server. If our api method CreateRun gets changed, e.g., in terms of its internal implementation, then create_run in SDK gets updated automatically. On the other hand, run_pipeline is a wrapper of api method CreateRun, and  run_pipeline sometimes might not be up-to-date (especially when we have a relatively big change in backend, e.g., introducing pipeline versions), unless we manually check/revise it to be up-to-date.  I personally prefer use create_run method.

Reply:
In the notebook, consider adding comments to call out the process of creating a run for a pipeline version.

Reply:
I'll add those comments in a separate PR to the example script, since the change is in a different repo than kubeflow/website.